renderdoc/api/replay/replay_enums.h

```diff
@@ -4716,6 +4716,30 @@ enum class AndroidFlags : uint32_t
 BITMASK_OPERATORS(AndroidFlags);
 DECLARE_REFLECTION_ENUM(AndroidFlags);
 
+DOCUMENT(R"(INTERNAL: The status of an API feature
+
+.. data:: NotAvailable
+
+  The feature is not available, e.g. because the extension that includes it is not enabled by the
+  application.
+
+.. data:: Disabled
+
+  The feature is available, but disabled
+
+.. data:: Enabled
+
+  The feature is enabled
+)");
+enum class FeatureStatus : uint32_t
+{
+  NotAvailable = 0,
+  Disabled,
+  Enabled,
+};
+
+DECLARE_REFLECTION_ENUM(FeatureStatus);
```

Comment:
This is a nice idea in principle but I think it's overkill for a feature like this, and I also want to avoid as much as possible having cap bits like this which the UI and other consumers using the python API then need to check. Instead any optional feature/functionality can simply be in the disabled/default state that exists when it's not used - whether it's not used or supported by the capture and driver or it's just disabled at the current event.

Also rather than making a separate review comment to be clear - I think it's fine to just have this state always have its column visible. It's not a huge invasive new section and there are several other features which are as-or-more obscure which are always visible e.g. line stipple. Viewport state is not something people look at often anyway, and the dimensions/offsets are still easy to see even with an extra column. If it were a huge imposition with lots of state then arguably it could be displayed conditionally like stream-out is on the geometry shader tab, but in that case there's not much value in distinguishing between not available and disabled.

Reply:
That's fair, done.

I also changed "Depth Clip Range" to "NDC Depth Range" to avoid any confusion with other depth-clip-related Vulkan extensions